server/src/main/java/org/opensearch/common/blobstore/BlobContainer.java

```diff
@@ -32,10 +32,14 @@
 
 package org.opensearch.common.blobstore;
 
+import org.opensearch.action.ActionListener;
```

Comment:
Changes in this file will be added as part of https://github.com/opensearch-project/OpenSearch/pull/8314. Reviewers can ignore these changes in this PR.